ClimateWatch-Vizzuality/indonesia-platform
# Add internalization + fetch translations from BE

This PR adds translations handler; the locale is expressed in the route and that is its single source of truth. Handling languages change is done by adding a location reducer (check `root-actions.js`)
- Modify router; added `onBeforeChange` handler
- Redirect to `/en` as default
- Prefix every route with `locale`
- Add language switcher to the navbar
- Integrate FE with BE in terms of translated content
- Updated slugs in the .yml files

To test: 
1. Clear your database from SectionContent: `SectionContent.destroy_all` (changed some slugs to make it easier on FE)
2. Run `rake db:add_content_management_records`
3. Run `rake db:add_initial_sections_content`

Play around :D Changing to Indonesian (what should be the correct language name btw?) leaves page without content because we don't have the translations ;p

Note:
The admin panel endpoint stays the same; without any locale, so localhost:3000/admin
